### ISSUE TITLE
3 dll are missing while installing Avatarify

### ISSUE THREAD

**Issue by RanjithAppireddy** [alievk/avatarify-desktop#57]
Today I'm downloaded Avatarify in my lap I'm not able to launch because of 3 dll are missing , I'm trying 3 time to re-install but same error was displayed , the missing dll are listed below 
1) MSVCP140.dll
2)VCRUNTIME140.dll
3)VCRUNTIME140_1.dll

can anyone please help me on this ASAP

**Comment by JohanAR** [comment]
Try googling it?

**Comment by Thenuad** [comment]
You'll need to (re)install C++ redistributables.

https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist?view=msvc-170
(I used the X64 | https://aka.ms/vs/17/release/vc_redist.x64.exe and it worked for me for all the missing .dlls).

**Note#1:** I actually downloaded the VC_redist.x64.exe from a different Microsoft location at first, and that file's size was below 20MB. That one **didn't** work. The one I provided the link above for (size around 24MB at this point) worked for me. I got to that MS page from a googled article (https://helpdeskgeek.com/windows-10/how-to-fix-a-vcruntime140-dll-is-missing-error-on-windows-10/... yeah, it says Windows 10 in the link, but it worked just fine on my Windows 11 sandbox a couple of hours ago).

**Note#2:** If you're using Nvidia RTX 30xx, though you might get those .dll dependencies taken care of by r(re)installing the C++ distributables, Avatarify will still be VERY sluggish. If you run into those issues, I recommend looking through the Issues section here (possibly, filtered in a similar fashion: https://github.com/alievk/avatarify-desktop/issues?q=is%3Aissue+is%3Aopen+slow).

Good luck! Let me know if you run into something else. I'm currently tweaking the thing myself and may have some insight and/or answers as I'm trying to follow a similar path ATM.

-V.
